common/religion/religions/arcane_group.txt

```diff
@@ -227,6 +227,7 @@
 	}
 
 	faiths = {
+		# Shalarism (Cult of Nightwell)
 		arcane_religion = {
```

Comment:
Not really. It's a religion of the highborne of Eldre'Thalas. They don't know of the Nightwell.

Reply:
Oh I thought that it was the nightborne one. Will remove comment.